assets/js/base/components/button/index.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { Button as WPButton } from 'wordpress-components';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+/**
+ * Component that visually renders a button but semantically might be `<button>` or `<a>` depending on the props.
+ *
+ * @todo Update cart and checkout button component. Cart and checkout should use base/components/button instead of cart-checkout/button.
+ */
+const Button = ( { className, showSpinner = false, children, ...props } ) => {
```

Comment:
This is a duplicate of the cart/checkout button - I logged a todo to migrate the cart/checkout to this one instead.

Reply:
It does indeed, however since this implements that button I think it's clearer to state that this is what will happen if used.